Backend/TimeTracker.Api/Controllers/ProjectController.cs

```diff
@@ -59,24 +59,37 @@ public async Task<GenericResponseDTO<Project>> GetProjectById(int id)
         /// </summary>
```

Comment:
Update the comments to match the change to the endpoint

Reply:
Updated Comments